mpcs/src/lib.rs

```diff
@@ -359,15 +359,79 @@ fn err_too_many_variates(function: &str, upto: usize, got: usize) -> Error {
 
 #[cfg(test)]
 pub mod test_util {
-
     use crate::{Evaluation, PolynomialCommitmentScheme};
     use ff_ext::ExtensionField;
     use itertools::{Itertools, chain};
-    use multilinear_extensions::mle::{DenseMultilinearExtension, MultilinearExtension};
-    use rand::{prelude::*, rngs::OsRng};
-    use rand_chacha::ChaCha8Rng;
+    use multilinear_extensions::{
+        mle::{DenseMultilinearExtension, MultilinearExtension},
+        virtual_poly_v2::ArcMultilinearExtension,
+    };
+    use rand::rngs::OsRng;
     use transcript::Transcript;
 
+    pub fn setup_pcs<E: ExtensionField, Pcs: PolynomialCommitmentScheme<E>>(
+        num_vars: usize,
+    ) -> (Pcs::ProverParam, Pcs::VerifierParam) {
+        let poly_size = 1 << num_vars;
+        let param = Pcs::setup(poly_size).unwrap();
+        Pcs::trim(param, poly_size).unwrap()
+    }
+
+    pub fn gen_rand_poly<E: ExtensionField>(
+        num_vars: usize,
+        base: bool,
```

Comment:
Didn't we have a PR where we moved away from using booleans here (and in similar situations)?

Reply:
Hmm, I guess that's happening in the follow up PR?

Reply:
@yczhangsjtu feel free either to resolve coversation (so i can merge) or change some code.